script/.build-deb/control

```diff
@@ -0,0 +1,15 @@
+Source: runc
+Maintainer: Open Containers <dev@opencontainers.org>
+Homepage: https://www.opencontainers.org/
+Vcs-Browser: https://github.com/opencontainers/runc
+Vcs-Git: git://github.com/opencontainers/runc.git
+
+Package: runc
+Architecture: linux-any
+Depends: ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Recommends: ca-certificates,
```

Comment:
Does it do any downloading from the Internet?  This particular `Recommend` might be misguided if not. :+1: